### DISCUSSION TITLE
Add a new log level or support only 'Last Attempted Proof' in logs

### DISCUSSION THREAD

**Discussion by wizwin** [Chia-Network/chia-blockchain#6198] [Ideas]
Many farmers just want to know only "Last Attempted Proof" information from the logs. GUI gives only 5 entries and many are interested what happened overnight, how is the system is performing etc.

My request is to:
1. Enhance this GUI to handle this in Latest Block Challenges
2. INFO logs gives a lot of output, we do not need that much information
- A new log level like INFO_FARM
- Enhance INFO level logs with a filter to tell I need only info on Last attempted proof or I need everything

**Comment by tiny0little** [comment]
best to post this to `Issues -> New Issue -> Make a Request`

**Comment by kevbo049** [comment]
Try this tactic.  I only want to know about how many of my plots were eligible for farming and the proofs found. So using NOTEPAD++,  I find the line in the log file that reads "2021-05-29T04:09:04.057 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming b2f1432152... Found 0 proofs. Time: 0.00100 s. Total 16 plots"  (the date and time portion will be different on your log file)  I then highlight "harvester chia.harvester.harvester: INFO" portion, then COPY, then click the "Search" menu, then "Find", and then PASTE into the Find What box and click "Find All in Current Document" button.  Here's what it shows me.  I do it for eligible plot and proofs found but you could paste whatever text within the log output file that you want to extract.  It's not perfect but it may skin that cat, though I've heard there's more that one way to do such things. 
<img width="1089" alt="Screenshot 2021-05-29 043732" src="https://user-images.githubusercontent.com/84846582/120065714-6a5b4500-c038-11eb-9d7e-02c5c3db54f8.png">

**Reply by wizwin** [reply]
Thank you @kevbo049. I usually use Notepad++ with RegEx ticked and search for "[1-9] plots were eligible". I am looking for a way to avoid all the extra logs and the manual task of searching these logs by setting the log level itself.

There is also a Windows PowerShell utility for Chia.

**Reply by MaximusChia** [reply]
Is a good workaround, anyways it needs to keep log writing a lot of stuff on disk, if only could write the harvester INFO related stuff.
